app.js

```diff
@@ -0,0 +1,22 @@
+
+let emailLogData = function () {
+
+  let emailLog = require("./emails.json");
+
+  const emailAddressCounts = emailLog.emails.reduce(function (prevVal, curVal) {
+    if (prevVal.hasOwnProperty(curVal.email)) {
+      prevVal[curVal.email] += 1;
+    }
+    if (!prevVal.hasOwnProperty(curVal.email)) {
```

Comment:
It's probably best to make this an else if, since then the code will try one or the other if case and not both. In the end, for this scenario it'll only run one of them but for the future, you may want to determine when it's best to run two if statements or an if and then an else if.